Fixed an issue to prevent from generating "No result returned" against an empty value. (#24539)
demisto/content: Packs/FiltersAndTransformers/ReleaseNotes/1_2_10.md

#### Scripts
##### AppendIfNotEmpty

> - Fixed an issue to prevent from generating "No result returned" against an empty value. 

Reviewer: ```suggestion
- Fixed an issue where *No result returned* to return an empty value. 
```
